diff --git a/openlayers.info b/openlayers.info
index 80dcad7..a9ead1c 100644
--- a/openlayers.info
+++ b/openlayers.info
@@ -2,7 +2,6 @@ name = "OpenLayers"
 description = "OpenLayers base API module"
 core = 7.x
 package = "OpenLayers"
-php = 5.3
 dependencies[] = libraries (>=2.1)
 dependencies[] = proj4js
 dependencies[] = ctools
diff --git a/openlayers.module b/openlayers.module
index a37cdfd..511b788 100644
--- a/openlayers.module
+++ b/openlayers.module
@@ -593,15 +593,28 @@ function openlayers_layer_types($reset = FALSE, $include_disabled = FALSE) {
   $layers = ctools_get_plugins('openlayers', 'layer_types');
 
   if (!$include_disabled) {
-    $layers = array_filter($layers, function($layer) {
-      return (!isset($layer->disabled) || !$layer->disabled);
-    });
+    $layers = array_filter($layers, 'openlayers_object_enabled');
   }
 
   return array_filter($layers, 'openlayers_layer_definition_check');
 }
 
 /**
+ * Confirm that an object is not disabled.
+ *
+ * @ingroup openlayers_api
+ *
+ * @param Object $object
+ *   An object -- a layer, behavior, style or map.
+ * 
+ * @return Boolean
+ *   Returns TRUE if the object is not disabled.
+ */
+function openlayers_object_enabled($object) {
+  return (!isset($object->disabled) || !$object->disabled);
+}
+
+/**
  * Menu loader for layer types.
  *
  * @ingroup openlayers_api
@@ -642,9 +655,7 @@ function openlayers_behaviors($reset = FALSE, $include_disabled = FALSE) {
   $behaviors = ctools_get_plugins('openlayers', 'behaviors');
 
   if (!$include_disabled) {
-    $behaviors = array_filter($behaviors, function($behavior) {
-      return (!isset($behavior->disabled) || !$behavior->disabled);
-    });
+    $behaviors = array_filter($behaviors, 'openlayers_object_enabled');
   }
 
   return array_filter($behaviors, 'openlayers_behavior_definition_check');
@@ -670,9 +681,7 @@ function openlayers_styles($reset = FALSE, $include_disabled = FALSE) {
   $styles = ctools_export_load_object('openlayers_styles', 'all', array());
 
   if (!$include_disabled) {
-    $styles = array_filter($styles, function($style) {
-      return !(isset($style->disabled) && $style->disabled);
-    });
+    $styles = array_filter($styles, 'openlayers_object_enabled');
   }
 
   return $styles;
@@ -754,9 +763,7 @@ function openlayers_maps($reset = FALSE, $include_disabled = FALSE) {
   $maps = ctools_export_load_object('openlayers_maps', 'all', array());
 
   if (!$include_disabled) {
-    $maps = array_filter($maps, function($map) {
-      return !(isset($map->disabled) && $map->disabled);
-    });
+    $maps = array_filter($maps, 'openlayers_object_enabled');
   }
 
   return $maps;
