diff --git a/modules/image/image.module b/modules/image/image.module
index c6a23f2..7159c9f 100644
--- a/modules/image/image.module
+++ b/modules/image/image.module
@@ -583,8 +583,10 @@ function image_styles() {
           $style['storage'] = IMAGE_STORAGE_DEFAULT;
           foreach ($style['effects'] as $key => $effect) {
             $definition = image_effect_definition_load($effect['name']);
-            $effect = array_merge($definition, $effect);
-            $style['effects'][$key] = $effect;
+            if ($definition) {
+              $effect = array_merge($definition, $effect);
+              $style['effects'][$key] = $effect;
+            }
           }
           $styles[$style_name] = $style;
         }
