diff --git a/omega/includes/omega.theme.inc b/omega/includes/omega.theme.inc
index 0ea7c4d..735ddb2 100644
--- a/omega/includes/omega.theme.inc
+++ b/omega/includes/omega.theme.inc
@@ -211,11 +211,7 @@ function omega_views_mini_pager($vars) {
   $tags = $vars['tags'];
   $element = $vars['element'];
   $parameters = $vars['parameters'];
-  $quantity = $vars['quantity'];
 
-  // Calculate various markers within this pager piece:
-  // Middle is used to "center" pages around the current page.
-  $pager_middle = ceil($quantity / 2);
   // current is the page we are currently paged to
   $pager_current = $pager_page_array[$element] + 1;
   // max is the maximum page number
