diff --git styleguide.styleguide.inc styleguide.styleguide.inc
index 5868055..b11b60f 100644
--- styleguide.styleguide.inc
+++ styleguide.styleguide.inc
@@ -52,6 +52,11 @@ function styleguide_styleguide() {
     'content' => styleguide_paragraph(3),
     'group' => t('Text'),
   );
+  $items['blockquote'] = array(
+    'title' => t('Blockquote'),
+    'content' => '<blockquote>' . styleguide_paragraph(2) . '</blockquote>',
+    'group' => t('Text'),
+  );
   $items['image-horizontal'] = array(
     'title' => t('Image, horizontal'),
     'content' => theme('image', styleguide_image('horizontal'), t('My image'), t('My image')),
